src/test/java/org/assertj/core/api/BDDSoftAssertionsTest.java

```diff
@@ -873,6 +873,17 @@ public void should_assert_using_assertSoftly() {
        .hasMessageContaining("blue");
   }
 
+  @Test
+  public void should_assert_using_assertSoftlyExecutables() {
```

Comment:
is there a difference with the test you have added in `SoftAssertionsTest` ? it there is not remove this one in favor of the other.